packages/eslint-plugin-react-native/README.md

```diff
@@ -0,0 +1,39 @@
+# @getluko/eslint-plugin-react-native
+
+This library was generated with [Nx](https://nx.dev).
+
+## Building
+
+Run `nx build eslint-plugin-react-native` to build the library.
+
+## Running unit tests
+
+Run `nx test eslint-plugin-react-native` to execute the unit tests via [Jest](https://jestjs.io).
+
+# @getluko/eslint-plugin-mobile-guidelines
+
+## Installation
+
+```
+yarn add @getluko/eslint-plugin-mobile-guidelines --dev
+```
+
+## Configuration
+
+`.eslintrc.js`
+
+```
+module.exports = {
+  plugins: ['@getluko/mobile-guidelines'],
```

Comment:
```suggestion
  plugins: ['@getluko/eslint-plugin-react-native],
```

Reply:
We can remove `eslint-plugin` at the beginning of the name. ESLint is translating it automatically.

`@getluko/react-native`